rebar.config

```diff
@@ -11,7 +11,7 @@
         {aeserialization, {git, "https://github.com/aeternity/aeserialization.git",
                            {ref, "3416ff5"}}},
         {aesophia, {git, "https://github.com/aeternity/aesophia.git",
-                    {tag, "v3.2.0"}}},
+                    {ref, "6a27c4a"}}},
```

Comment:
That's not the newest

Reply:
It is new enough for this application :-) We will sync them properly when it is time for release.